src/main/java/control/TimelineController.java

```diff
@@ -53,10 +53,9 @@ public void addCameraShot(int cameraIndex, String name, String description,
         CameraShot newShot = new CameraShot(name,description, startCount, endCount);
         this.scriptingProject.getCameraTimelines().get(cameraIndex).addShot(newShot);
         CameraShotBlock shotBlock = new CameraShotBlock(newShot.getInstance(), cameraIndex,
-                                                        rootPane.getRootCenterArea(),
-                                                        startCount, endCount);
+                rootPane.getRootCenterArea(), startCount, endCount,
+                this.rootPane.getRootCenterArea().getGrid());
```

Comment:
Why is the timeline grid being passed to the constructor if the RootCenterArea is already being passed?